packages/minizip.rb

```diff
@@ -1,40 +1,34 @@
 require 'package'
+require_relative 'zlibpkg'
 
 class Minizip < Package
   description 'Minizip is a simple package to zip/unzip files, from zlib.'
-  homepage 'https://www.zlib.net/'
-  @_ver = '1.2.11'
+  homepage Zlibpkg.homepage
+  @_ver = '1.2.13'
   # When upgrading minzip, be sure to upgrade zlibpkg in tandem.
-  version @_ver
-  license 'zlib'
+  version Zlibpkg.version
+  license Zlibpkg.license
   compatibility 'all'
-  source_url "https://www.zlib.net/zlib-#{@_ver}.tar.gz"
-  source_sha256 'c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1'
+  source_url Zlibpkg.source_url
+  source_sha256 Zlibpkg.source_sha256
```

Comment:
Are these supposed to be replaceable parameters?

Reply:
what do you mean?

Reply:
See my other comment 
https://github.com/chromebrew/chromebrew/pull/7557#discussion_r1006138811